test/compflags/checks/checkModByZero.chpl

```diff
@@ -13,47 +13,47 @@ record R {
 }
 
 /* Define this so that %= is automatically defined */
-proc %(lhs: R, rhs: R) return new R(lhs.x % rhs.x);
+operator R.%(lhs: R, rhs: R) return new R(lhs.x % rhs.x);
 
 var rec1 = new R(1),
     rec0 = new R(0);
 
 select testParamCases {
-  //proc %(param a: int(?w), param b: int(w)) param
+  //operator R.%(param a: int(?w), param b: int(w)) param
   when 1 do
     writeln(10 % 0);
 
-  //proc %(param a: uint(?w), param b: uint(w)) param
+  //operator R.%(param a: uint(?w), param b: uint(w)) param
   when 2 do
     writeln(10 % 0);
 
-  //proc %(a: uint(64), param b: uint(64))
+  //operator R.%(a: uint(64), param b: uint(64))
   when 3 do
     writeln(unum % 0);
 }
 
 select testExecCases {
-  //proc %(a: int(?w), b: int(w))
+  //operator R.%(a: int(?w), b: int(w))
   when 0 do
     writeln(num % denom);
 
-  //proc %(a: uint(?w), b: uint(w))
+  //operator R.%(a: uint(?w), b: uint(w))
   when 1 do
     writeln(unum % udenom);
 
-  //proc %=(ref lhs:int(?w), rhs:int(w))
+  //operator R.%=(ref lhs:int(?w), rhs:int(w))
   when 2 do
     num %= denom;
 
-  //proc %=(ref lhs:uint(?w), rhs:uint(w))
+  //operator R.%=(ref lhs:uint(?w), rhs:uint(w))
   when 3 do
     unum %= udenom;
 
-  //proc %=(ref lhs, rhs)
+  //operator R.%=(ref lhs, rhs)
```

Comment:
Except for this comment, which should be `operator R.%`. We could type the args, but I don't think that matters too much.